addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -370,7 +370,7 @@ export class HtmlField extends Component {
             // Ensure the focus is in the editable document
             // before inserting the <t> element.
             this.wysiwyg.focus();
-            let dynamicPlaceholder = "object." + chain.join('.');
+            let dynamicPlaceholder = "object." + chain;
```

Comment:
@dmo-odoo After fixing main error of the task there was another traceback occured and this line if fixing that another traceback.

Reply:
I think it should have been in its own commit because it is not fixing the same issue as the original commit. However, I'm going to r+ it anyway because we have had multiple customer tickets about this issue so I want it to be fixed quickly. Next time, please make another commit if it is a fix for something else, thanks ! :)